C/Clipper2/build_tarballs.jl

```diff
@@ -41,4 +40,4 @@ dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"11.1.0")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"13.1.0")
```

Comment:
GCC 13 [isn't even available](https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/master/src/Rootfs.jl#L392-L406)
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"11.1.0")
```

Reply:
Oops, didn't realise that version is not available... Why do you propose v11.1.0 instead of v12.1.0?

Reply:
https://github.com/JuliaPackaging/Yggdrasil/blob/5fb343ac5c89a09866678eedcedc1daccb33bebb/CONTRIBUTING.md#compatibility-tips

Reply:
> _As a general remark, when using GCC as compiler (which is default when targeting Linux and Windows platforms) try to use the oldest versions that is able to compile your code (...) In any case, avoid using GCC 11+ when building C++, as that would be incompatible with Julia v1.6 (...)_

Thanks!